File: module-load-balance-gateway/src/main/java/com/xmmufan/permission/aspect/UpdatePermissionCache.java
package com.xmmufan.permission.aspect;

import java.lang.annotation.*;

/**
 * @author: Mr.Hergua | 黄源钦
 * @version: 1.0
 * @Date 2018/11/24
 * <p>
 *     更新权限缓存注解
 * </p>
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
@Documented
public @interface UpdatePermissionCache {
}

File: module-load-balance-gateway/src/main/java/com/xmmufan/permission/aspect/UpdatePermissionCache.kt
package com.xmmufan.permission.aspect

/**
 * @author: Mr.Hergua | 黄源钦
 * @version: 1.0
 * @Date 2018/11/24
 *
 *
 * 更新权限缓存注解
 *
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
@Retention(AnnotationRetention.RUNTIME)
@MustBeDocumented
annotation class UpdatePermissionCache
